examples/kuadrant/combined-loadbalanced-dns.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: kuadrant.io/v1alpha1
```

Comment:
no, they're just useful DNS Records for testing the various use-cases (instead of going through all the mess of setting up kuadrant-operator and creating a DNS Policy to output these same DNS Records).